cypress/support/commands.ts

```diff
@@ -11,6 +11,7 @@ Cypress.on('uncaught:exception', (err, _) => {
   // we expect a 401 error when logging in with DEX's APIs, so ignore that but fail for all other console errors.
   // we also can ignore AbortErrors for when network requests are terminated early.
   return !(
+    err.message.includes('unauthorized access') ||
```

Comment:
for anyone wondering why this needed to be added, Subir and I tried for several hours to avoid having to add this exception but couldn't find an alternate solution. Basically between the two tests, cypress clears the session cookies and local storage cache and any queued API requests from the end of the first test are made without the API token, causing them to return 401. It was very inconsistent and we couldn't find a solution with cy.intercept that would help solve it. So instead we're asking cypress to ignore any of these `unauthorized access` responses.